util/travis/before_install.sh

```diff
@@ -9,7 +9,35 @@ if [ $CC = "clang" ]; then
 	sudo apt-get install clang
 fi
 sudo apt-get install p7zip-full
-if [ $WINDOWS = "no" ]; then
+if [ $ANDROID = "yes" ]; then
+	
+	printf "Generating swap..."
```

Comment:
The travis build machines should have more than enough RAM, why do you need swap?